clang-tools-extra/docs/clang-tidy/checks/readability/container-data-pointer.rst

```diff
@@ -3,13 +3,9 @@
 readability-container-data-pointer
 ==================================
 
-Finds cases where code could use ``data()`` rather than the address of the
-element at index 0 in a container. This pattern is commonly used to materialize
-a pointer to the backing data of a container. ``std::vector`` and
-``std::string`` provide a ``data()`` accessor to retrieve the data pointer which
-should be preferred.
+Finds cases where code references the address of the element at index 0 in a container and replaces them with calls to ``data()`` or ``c_str()``.
```

Comment:
Wrap on 80 column,
And first sentence of documentation should be in sync with class description.

Reply:
Fixed, thanks :)

Reply:
Still not wrapped on 80 collumn